backend/kernelCI_app/views/treeTestsView.py

```diff
@@ -44,19 +44,19 @@ def __paramsFromBootAndFilters(self, commit_hash, path_param, filter_params, par
             # TODO: 'boot' and 'boot.', right now is only using 'boot.'
             path_filter = "AND t.path LIKE %s"
             params.append(f'{path_param}%')
-            target_field = 'bootStatus'
         else:
             path_filter = ""
-            target_field = 'testStatus'
 
         for f in filter_params.filters:
-            if f['field'] != target_field:
-                continue
             value = f['value']
-            value_list = value if isinstance(value, list) else [value]
-            placeholders = ",".join(['%s' for i in value_list])
-            path_filter += "AND t.status IN (" + placeholders + ")"
-            for value in value_list:
+            value_is_list = isinstance(value, list)
+            if value_is_list:
+                placeholder = ",".join(['%s' for i in value]) if value_is_list else '%s'
+                path_filter += f" AND t.{f['field']} IN ({placeholder})"
+                params.extend(value)
+            else:
+                comparison_op = filter_params.get_comparison_op(f, 'raw')
+                path_filter += f" AND t.{f['field']} {comparison_op} %s"
```

Comment:
Please add a TODO  to swap those string interpolation with parametrized queries since this doesn't get sanitized and it is a vulnerability

Reply:
I would say that we should sanitize the input instead of adding a TODO

Reply:
I agree, it is fairly easy, just use a dictionary as a parameter and use `%(key)s` structure like I did in my PR, instead of an `f-string`

https://github.com/kernelci/dashboard/pull/236/files

Reply:
The thing is, I also need to pass the parameters as a list here and we cant do booth, I look for alternatives

Reply:
I cant use a dict and a list as parameters at the same time, i will look for an alternative...

Reply:
I can check the ['field'] to make sure it is valid